docs/application/dotnet/guides/applications/service_application.md

```diff
@@ -0,0 +1,213 @@
+| Sensor (context)               | Processing context data from the sensors, such as gesture | [Tizen.Sensor](/application/dotnet/api/TizenFX/latest/api/Tizen.Sensor.html) | `sensor`                                 |
+| IoT Communication/Connectivity | Communicating between external devices in the background (such as Wi-Fi and Bluetooth) | [Tizen.Network.WiFi](/application/dotnet/api/TizenFX/latest/api/Tizen.Network.WiFi.html) <br> [Tizen.Network.Bluetooth](/application/dotnet/api/TizenFX/latest/api/Tizen.Network.Bluetooth.html) | `iot-communication`                      |
+
+
+> [!NOTE]
+> Since Tizen 4.0, even if the background network category is declared, the running application stops if the network is not connected.
+
+### Describe background category
+
+An application with a background running capability must declare the background category in its manifest file:
+
+```
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns="http://tizen.org/ns/packages" api-version="2.4" package="org.tizen.test" version="1.0.0">
+   <ui-application appid="org.tizen.test" exec="text" type="capp" multiple="false" taskmanage="true" nodisplay="false">
+      <icon>rest.png</icon>
+      <label>rest</label>
+      <!--For API version 2.4 and higher-->
+      <background-category value="media"/>
+      <background-category value="download"/>
+      <background-category value="background-network"/>
+   </ui-application>
+   <service-application appid="org.tizen.test-service" exec="test-service" multiple="false" type="capp">
+      <background-category value="background-network"/>
+      <background-category value="location"/>
+   </service-application>
+</manifest>
+```
+
+## Manage the service application
+
+The following code snippets show the service application backbone generated from the Tizen templates.
+The service template creates callback stubs to fill. Service application type is defined in the `Tizen.Applications` namespace, so it should be included at the top of the file. 
+
+```csharp
+using Tizen.Applications;
+```
+
+The `ServiceApplication` has no UI, so `OnPause()` and `OnResume()` are not defined. Besides of that it defines similar callbacks to other app types like `OnCreate()` and `OnTerminate()` lifecycle callbacks, and `OnLowBattery()`, `OnLowMemory()`, `OnLocaleChanged()` system callbacks.
+
+`OnCreate()` method is used to take necessary actions, before the main event loop starts. Place the initialization code here:
+
+```csharp
+        protected override void OnCreate()
+        {
+            base.OnCreate();
+            
+            //Initialization
+        }
+```
+
+`OnAppConrolReceived()` callback is the most important callback since it is responsible for data exchange between the service app and other applications:
```

Comment:
```suggestion
`OnAppControlReceived()` callback is the most important callback since it is responsible for data exchange between the service app and other applications:
```